INSTALL.md

```diff
@@ -52,18 +50,28 @@ This will use `/opt/circleci` as the base install location.
 
 First, set one of these variables as appropriate for for your installation target.
 
+#### For Linux AMD64
 ```bash
-# For Linux x86_64
 platform=linux/amd64
-# For Linux ARM
+prefix=/opt/circleci
+```
+
+#### For Linux ARM
+```bash
 platform=linux/arm64
+prefix=/opt/circleci
+```
+
+#### For macOS AMD64
+```bash
+platform=darwin/amd64
+prefix=/usr/local/bin
```

Comment:
any reason to prefer `/usr/local` over `/opt/circleci` on Mac?

I generally like opt for this sort of thing as you know it's only your own stuff

Reply:
It's pretty typical to install stuff here on macOS, and very unusual to put in `/opt`